package.json

```diff
@@ -40,6 +40,25 @@
   "engines": {
     "node": ">=14.0.0"
   },
+  "type": "module",
+  "types": "./dist/src/index.d.ts",
+  "files": [
+    "src",
+    "dist/src",
+    "!dist/test",
+    "!**/*.tsbuildinfo"
+  ],
+  "exports": {
+    ".": {
+      "import": "./dist/src/index.js"
```

Comment:
I followed other libp2p projects, and allowed exporting as ESM modules only.

Reply:
sounds good to me